### PR TITLE
Developer

### PR DESCRIPTION
**Pull request by TashfeenRao**

### CODE REVIEW

File: css/media.css [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+  .black-footer {
+    height: 97px;
+  }
+}
+
+@media screen and (min-width: 750px) {
+  .nav-mob {
+    display: none;
+  }
+}
```
Comment (suretrust) [inline css/media.css:476]: There are some problems with your media queries. Some overflow (horizontal navbar) appears between 750px and 820px as well as between 1024px and 1145px screens.

vaertical

File: index.html [hunk 2]
```diff
@@ -0,0 +1,995 @@
...
+            <li><a href="#">AMAs</a></li>
+            <li><a href="#">Spaces</a></li>
+            <li><a href="#">Terms & Conditions</a></li>
+          </ul>
+          <ul class="dark-nav-right-links">
+            <li><a href="#">About</a></li>
+            <li><a href="#">Team</a></li>
+            <li><a href="#">Advertise</a></li>
+            <li><a href="#">Jobs</a></li>
+            <li><a href="#">Contact</a></li>
```
Comment (suretrust) [inline index.html]: On medium screen size, the navigation bar is different from the one in your design.

**Yours**
nav2



**Original**
nav

File: index.html [hunk 3]
```diff
@@ -0,0 +1,995 @@
...
+      <main class="main">
+        <section class="grid-articles">
+          <a href="#" class="latest-news-title">
+            <h1>Latest News ></h1>
+          </a>
+          <div class="grid-box">
+            <a class="letest-news-cover" href=""
+              ><img
+                src="https://images.unsplash.com/photo-1518770660439-4636190af475?ixlib=rb-1.2.1&auto=format&fit=crop&w=750&q=80"
+                alt=""
```
Comment (suretrust) [inline index.html]: On large screen, the latest news should have 4 rows.

**Original**
latest2



**Yours**
latest

File: index.html [hunk 4]
```diff
@@ -0,0 +1,995 @@
...
+                ><img
+                  src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg"
+                  alt=""
+              /></a>
+            </li>
+          </ul>
+          <ul class="sm-nav-right">
+            <li>
+              <a href="#"><i class="fa fa-search"></i></a>
+            </li>
```
Comment (suretrust) [inline index.html]: This section on medium screen looks different from your design.

**Original**
head2


**Yours**
head

Comment (suretrust) [reply index.html]: On small screen, the navbar has the hamburger symbol.

**Yours**
nav

**Original**
nav2

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi team!

### Amazing job thus far, :raised_hands:  just a few suggestions to wrap it up. Please see my comments on your code.

**Kindly submit for another code review once these issues have been fixed.** _Happy coding!_ :tada:

**Comment by shshamim63** [comment]
## Status :red_circle: [!!Required Changes!!] :red_circle: 
___
Hi @TashfeenRao & @mekinus :wave: ,
Great job :clap: . I can see you have covered some issues that has been raised by the previous TSE. But I think there are some issues that you need to fix too. Well I am sure you guys will complete it. 
#### Please follow the instructions below :point_down: to declare this project as complete.
___
## :spades: [Mandatory Changes] :spades: 
___
### :large_blue_diamond:  Navbar :large_blue_diamond: 
___
####  Medium screen
___
1. Well on the medium screen size, on the navbar the TNW logo should appear. But I can not see it on your project.
##### Your Project--->
tnwlogomedium
##### Actual Webpage--->
actual logo
2. This is project is about responsive web design. So your elements should interact with respect to the window size. And a feature of this is, it should contain all the element without overflowing. But I can see it. On screen- size `754px-850px` the page is overflowing.
overflow
overflow2
It is because of the fixed width. For an example width: 250px;
 ___
####  Mobile screen
___
1. Items are overflowing. Think about the width. What do you think about using the width in percentage(%)? For an example width: 45%;
footer
footerover2
___
#### Please fix the above mentioned issue and send another code review request.
___
## :herb: Happy Coding :herb: 
### Shakhawat Hossain
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

**Review by Redvanisation — CHANGES_REQUESTED** [review]
# Changes required  ♻️ 

### Hello team  🌟

**Well done implementing the changes, but we need to fix the issues below before moving on ⬇️** 

- On medium screen sizes and when the logo appears on the navbar the big logo should disappear, on your page, they are both showing at the same time, please fix that.

Screen Shot 2019-11-24 at 7 05 20 PM

### _After making the required changes please submit a new code review request_

### Good luck! 🍀

**Review by imhta — APPROVED** [review]
# Project Approved 👍 

### Hello @TashfeenRao @mekinus , 👋 You have done great job with this project. 👏 



----
#### Don't forgot to sumbit project completion form
 **Pon Muthu Selvam N**
Feel free contact me 
[Github //@imhta](https://github.com/imhta) | [Twitter //@nponmuthuselvam](https://twitter.com/ponmuthuselvam)

**Pull request merged**